lib/optimizely/project_config_schema.js

```diff
@@ -45,34 +48,41 @@ module.exports = {
               "type": "object",
               "properties": {
                 "entityId": {
-                  "type": "string"
+                  "type": "string",
+                  "required": true
                 },
                 "endOfRange": {
-                  "type": "integer"
+                  "type": "integer",
+                  "required": true
                 }
               },
               "required": [
```

Comment:
Are these entries still needed if we're also marking `"required": true` at the entity level?